src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -63,9 +64,14 @@
    {:id :about :label (i18n/label :t/about) :accessibility-label :about}])
 
 (defn view
-  []
-  (let [top          (safe-area/get-top)
-        selected-tab (reagent/atom (:id (first tabs-data)))]
+  [account-address]
+  (let [accounts        (rf/sub [:profile/wallet-accounts])
```

Comment:
I think we should create another sub to get this `account` info and it should be based off the id. I believe all of this information should be handled in there too. e.g the balance etc

Reply:
Also I think subscriptions should be inside the renderer

Reply:
Done